pkg/framework/test/etcd.go

```diff
@@ -45,15 +43,13 @@ type SimpleSession interface {
 type simpleSessionStarter func(command *exec.Cmd, out, err io.Writer) (SimpleSession, error)
 
 // NewEtcd constructs an Etcd Fixture Process
-func NewEtcd(pathToEtcd, etcdURL, etcdPeerURL string) *Etcd {
+func NewEtcd(pathToEtcd string) *Etcd {
```

Comment:
I was going to note on the other commit before that I was a little bit concerned about the handling of configurations here, I'm not sure I agree with the direction.

Reply:
I am not sure what you want to tell me with the comment. Is it any of those:
- All configuration should be given to the constructor.
- Not even the path to the binary should be given at construction time but configured later.

My line of thought was something like:
- If I have, for example, 2 different etcd binaries, my `Etcd`s will be different things. Therefore `Etcd` will be constructed with a specific binary.
- The configuration might change, but as long as the binary stays the same, it is the very same `Etcd`. The configuration might also change between different runs. Therefore to place to hand that in is actually at start time.